cmd/dp-hierarchy-builder/main.go

```diff
@@ -41,28 +42,42 @@ func main() {
 	log.Event(ctx, "loaded config", log.INFO, log.Data{"cfg": cfg})
 
 	kafkaBrokers := cfg.KafkaAddr
-	cgChannels := kafka.CreateConsumerGroupChannels(true)
+
+	kafkaOffset := kafka.OffsetNewest
+
+	if cfg.KafkaOffsetOldest {
+		kafkaOffset = kafka.OffsetOldest
+	}
+
+	cgConfig := &kafka.ConsumerGroupConfig{
+		Offset:       &kafkaOffset,
+		KafkaVersion: &cfg.KafkaVersion,
+	}
+
+	cgChannels := kafka.CreateConsumerGroupChannels(1)
```

Comment:
Can you create a const or variable for `1` literal please? Using a well named variable for values like this gives additional context without having to step into the function to see what the mean/do.